### PR TITLE
STYLE: Prefer c++11 'using' to 'typedef'

### PR DESCRIPTION
**Pull request by hjmjohnson**

### CODE REVIEW

File: Modules/Filtering/FastMarching/test/itkFastMarchingImageFilterBaseTest.cxx [hunk 1]
```diff
@@ -87,7 +87,7 @@ int itkFastMarchingImageFilterBaseTest( int , char * [] )
   // Done outside the helper function in the test because GCC is limited
   // when calling overloaded base class functions.
   const unsigned int Dimension = 2;
-  typedef float                               PixelType;
+  using PixelType = float;
   typedef itk::Image< PixelType, Dimension >  ImageType;
```
Comment (dzenanz) [inline Modules/Filtering/FastMarching/test/itkFastMarchingImageFilterBaseTest.cxx]: This `typedef` remains?

File: Modules/Filtering/FastMarching/test/itkFastMarchingImageFilterBaseTest.cxx [hunk 2]
```diff
@@ -87,7 +87,7 @@ int itkFastMarchingImageFilterBaseTest( int , char * [] )
...
 
   typedef itk::FastMarchingImageFilterBase< ImageType, ImageType >
```
Comment (dzenanz) [inline Modules/Filtering/FastMarching/test/itkFastMarchingImageFilterBaseTest.cxx]: This `typedef` remains?

File: Modules/Filtering/FastMarching/test/itkFastMarchingImageFilterBaseTest.cxx (hunk 1 above)
Comment (hjmjohnson) [reply Modules/Filtering/FastMarching/test/itkFastMarchingImageFilterBaseTest.cxx]: Fixed.

**Comment by hjmjohnson** [comment]
@dzenanz Please merge if you approve.

**Pull request merged**